metron-analytics/metron-profiler/src/main/config/profiler.properties

```diff
@@ -39,4 +39,4 @@ profiler.hbase.flush.interval.seconds=30
 
 kafka.zk=node1:2181
 kafka.broker=node1:6667
-kafka.start=WHERE_I_LEFT_OFF
+kafka.start=UNCOMMITTED_EARLIEST
```

Comment:
This needs documentation.  We had our own abstractions around the kafka offsets which we no longer need because the new spout has better ones.